packages/paste-website/src/components/tokens-list/token-card/token-example/BoxExample.tsx

```diff
@@ -19,6 +23,7 @@ export const BoxExample: React.FC<BoxExampleProps> = ({
     <Box
       aria-hidden="true"
       backgroundColor={backgroundColor}
+      style={{backgroundColor: highlightColor}}
```

Comment:
Question: can this be a regular style prop? Noticed a few usages of inline styles throughout the PR and wondering why we had to use those